code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -140,6 +139,53 @@
 
 //Spooky special loot
 
+//Rod of Asclepius
+/obj/item/rod_of_asclepius
+	name = "Rod of Asclepius"
+	desc = "A wooden rod about the size of your forearm with a snake carved around it, winding it's way up the sides of the rod. Something about it seems to inspire in you the responsibilty and duty to help others."
+	icon = 'icons/obj/lavaland/artefacts.dmi'
+	icon_state = "asclepius_dormant"
+	var/activated = FALSE
+
+/obj/item/rod_of_asclepius/attack_self(mob/user)
+	if(!activated)
+		if(iscarbon(user))
```

Comment:
discussed this last night on discord, replace long if small else with small if big else

so if(activated) return and if(!iscarbon(user)) blah blah blah return then post the rest

Reply:
K